frontend-react/src/shared/Tile/Tile.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import DOMPurify from "dompurify";
+
+import { SectionProp } from "../../pages/home/Sections/Section";
+
+export interface ItemProp {
+    method?: number;
+    title?: string;
+    icon?: string;
+    img?: string;
+    imgAlt?: string;
+    imgClassName?: string;
+    summary?: string;
+    items?: { title?: string; summary?: string }[];
+}
+
+export const Tile = ({
+    section,
+    item,
+}: {
+    section: SectionProp;
+    item: ItemProp;
+}) => {
+    let cleanSummaryHtml = DOMPurify.sanitize(item!.summary!);
+    const totalItems = section.items?.length || 0;
+    let gridColValue = 12 / totalItems;
+    const styleItems = `tablet:grid-col-${gridColValue} margin-bottom-0`;
+
+    return (
+        <div className={styleItems}>
+            {item.img && (
+                <img
+                    data-testid="img"
+                    src={item.img}
+                    alt={item.imgAlt}
+                    className={item.imgClassName}
+                />
+            )}
+            {item.title && (
+                <h3
```

Comment:
Still do not like hardcoding the header level here. As a temporary middle ground, can we accept a headerLevel prop  as a string that will override this (don't limit the strings to only h1-h6, just accept string for the typing)?

Reply:
Sorry, meant to fix this.